index.html

```diff
@@ -65,19 +65,20 @@ <h1>Hello, world!</h1>
       <!-- Example row of columns -->
       <div class="row">
         <div class="col-md-4">
-          <h2>Heading</h2>
-          <p>Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui. </p>
-          <p><a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
+          <h2>The Star</h2>
```

Comment:
This looks like a good transition to something for the site. I too add just a small sample of the poem to the homepage but I think we could use yours since you set up pages to go with it. Great work.